### ISSUE TITLE
PHP7 + Mysqli support

### ISSUE THREAD

**Issue by eduh** [thexerteproject/xerteonlinetoolkits#830]
Is there a roadmap available for Xerte so that it works with PHP7.x and Mysqli or PDO ?

**Comment by eduh** [comment]
When I try to install the latest dev version it the install  says that the mysql extension is required. This extension is obsolete in php7 so how to continue?

**Comment by torinfo** [comment]
Allthough the link for  more information still points to how to install the old mysql extension (fixed now), the setup actually tests for the availabillity of the PDO and Mysql PDO extensions.

**Comment by eduh** [comment]
Thank you. 
I dived into the `static function MysqlCheck() ` and found out the check is stricktly on PDO. I have a standalone apache \php\mysql stack and found out I was too lazy to enable the mysql pdo driver. So sorry for that. Problem solved when I did enable the driver. 
Perhaps the install manual could be extended with a small notification that when using a standalone stack the PDO driver must be enabled to complete the setup.
